swim/member.go

```diff
@@ -107,6 +110,8 @@ func statePrecedence(s string) int {
 		return 2
 	case Leave:
 		return 3
+	case Tombstone:
```

Comment:
There is a lot of work to do to make this a propper type and constant because the names are directly used in logging and in the marshaling and un-marshaling from and to json. I'm very much in favor of doing it, but in a contained PR because it will on its own already touch a lot of code which has nothing todo with what we are adding here.

Lets put a task on the board and properly plan it.